src/common/components/Button/Button.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { FC } from "react";
+import styles from "./Button.module.scss";
+
+interface ButtonProps {
+  text: string;
+  type?: "button" | "submit";
+  buttonClick?: () => void;
```

Comment:
It's button component, so you can skip button prefix for styles, isDisabled - as for text,type

Reply:
Done - remove prefixes from button component